Makefile

```diff
@@ -36,10 +35,7 @@ docker-compile:
 # For cases where we're building from local
 docker-build:
 	docker build --rm -t ${IMAGE} rootfs
-
-# Push to a registry that Kubernetes can access.
-docker-push:
-	docker push ${IMAGE}
+	docker tag -f ${IMAGE} ${MUTABLE_IMAGE}
```

Comment:
I was just copying this from existing files. Let me know if you want me to remove -f tag in all other PRs that I have opened along with this one